h2o-genmodel-extensions/xgboost/src/main/java/hex/genmodel/algos/xgboost/XGBoostMojoModel.java

```diff
@@ -42,7 +52,7 @@ public static ObjectiveType fromXGBoost(String type) {
   public int[] _catOffsets;
   public boolean _useAllFactorLevels;
   public boolean _sparse;
-  public String _featureMap;
+  public String[] _featureMap;
```

Comment:
Can you please explain why does this need to be changed? Any change like that can break backward compatibility.